streams/src/test/java/org/apache/kafka/streams/state/internals/CachingKeyValueStoreTest.java

```diff
@@ -339,12 +366,24 @@ public void shouldThrowIfTryingToDoRangeQueryOnClosedCachingStore() {
         store.range(bytesKey("a"), bytesKey("b"));
     }
 
+    @Test(expected = InvalidStateStoreException.class)
```

Comment:
Use `assertThrows` -- we've been (slowly) migrating away from `@Test(expected)` in the Streams tests

Reply:
migrated the whole class. will apply this to the other PRs.